x/sequencer/types/messages.go

```diff
@@ -67,29 +69,36 @@ func (msg *MsgCreateSequencer) GetSignBytes() []byte {
 func (msg *MsgCreateSequencer) ValidateBasic() error {
 	_, err := sdk.AccAddressFromBech32(msg.Creator)
 	if err != nil {
-		return sdkerrors.Wrapf(sdkerrors.ErrInvalidAddress, "invalid creator address (%s)", err)
+		return errorsmod.Wrapf(errortypes.ErrInvalidAddress, "invalid creator address (%s)", err)
 	}
 
 	// public key also checked by the application logic
 	if msg.DymintPubKey != nil {
```

Comment:
why we allow `DymintPubKey` to be nil in the msg?
we can set it to `nullable = false` in the proto file, to require it on the `MsgCreateSequencer`
I'll open separate issue for it